2015-06-17 10:13:54,040 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,041 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:13:54,129 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,129 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:13:54,215 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,216 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:13:54,301 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,302 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:13:54,389 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,389 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:13:54,478 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:13:54,479 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:14:10,139 my_lib.py[line:280] ERROR 模块不存在，无法获取信息!
2015-06-17 10:14:10,139 my_lib.py[line:280] ERROR [1;31;40m**Error**[0m 无法获取主机信息.

2015-06-17 10:14:30,230 my_lib.py[line:277] INFO [1;32;40m**Success**[0m 远程主机vlan ip部署成功!

2015-06-17 10:14:30,231 my_lib.py[line:277] INFO [1;32;40m**Success**[0m 配置vlan ip 成功.

2015-06-17 10:14:30,257 my_lib.py[line:277] INFO [1;32;40m**Success**[0m 发送邮件 成功.

2015-06-17 10:15:17,157 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.139.20.209  网络畅通.

2015-06-17 10:15:17,158 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.149.224.34  网络畅通.

2015-06-17 10:15:17,158 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.139.224.34  网络畅通.

2015-06-17 10:15:17,159 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.149.20.209  网络畅通.

2015-06-17 10:17:40,208 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.139.20.209  网络畅通.

2015-06-17 10:17:40,208 my_lib.py[line:277] INFO [1;32;40m**Success**[0m vlan ip: 10.149.224.34  网络畅通.

2015-06-17 10:17:40,209 my_lib.py[line:277] INFO [1;32;40m**Success**[0m vlan ip: 10.139.224.34  网络畅通.

2015-06-17 10:17:40,209 my_lib.py[line:277] INFO [1;32;40m**Success**[0m ip: 10.149.20.209  网络畅通.

2015-06-17 10:18:14,097 my_lib.py[line:277] INFO [1;32;40m**Success**[0m machine ip: 10.139.20.209  网络畅通.

2015-06-17 10:18:14,098 my_lib.py[line:277] INFO [1;32;40m**Success**[0m vlan ip: 10.149.224.34  网络畅通.

2015-06-17 10:18:14,098 my_lib.py[line:277] INFO [1;32;40m**Success**[0m vlan ip: 10.139.224.34  网络畅通.

2015-06-17 10:18:14,099 my_lib.py[line:277] INFO [1;32;40m**Success**[0m machine ip: 10.149.20.209  网络畅通.

2015-06-17 10:18:22,917 my_lib.py[line:277] INFO 

[1;32;40m**Success**[0m Good Luck Boys.........

